scalafmt-tests/src/test/resources/default/SearchState.stat

```diff
@@ -455,8 +455,8 @@ EventsResponse.fromJson(json, run).success.value.events should be(
     ))
 <<< react #458
   val TodoApp = ReactComponentB[Unit]("TodoApp")
-    .initialState(State(Nil, ""))
-    .renderS(($, s) =>
+    .initialState(State(Nil, "")).renderS(
```

Comment:
Is this an desirable change? This case looks like a regression compared to before.

Reply:
Whoops, for some reason I didn't see that. Looks great!